api/src/api/session.js

```diff
@@ -30,6 +30,51 @@ router.get(
   }),
 );
 
+//get all displayed sessions on the home page
+router.get(
+  '/displayed',
+  errorWrap(async (req, res) => {
+    let activeSessions = await Session.find({ active: 'true' });
+    let max = 10;
+    //if there are >= max active sessions, all will be displayed
```

Comment:
Comments should have a space between the `//` and the first letter. And the first letter should be capitalized:
`// For example like this`
Also make sure to edit this (and other) comments with the new maxDisplayed variable name

Reply:
Gotcha!